src/settings/ImportPage.tsx

```diff
@@ -242,7 +236,17 @@ export const ImportPage = () => {
       <h1><i className="fas fa-upload"></i> Import Data</h1>
       <Row>
         <Col lg={8}><ImportPreview importData={getData()} triggerRender={triggerRender} /></Col>
-        <Col lg={4}>{getAction()}</Col>
+        <Col lg={4}>
+          {people.length === 0
```

Comment:
I know this is a matter of preference, but I prefer to call functions rather than use inline code for scenarios like these just to improve code readability.  Doesn't matter too much when it's a 9 line block of code, but when it gets to be 20+ lines it's hard to see the big picture layout.